### DISCUSSION TITLE
Folder .chia is TO BIG

### DISCUSSION THREAD

**Discussion by nicolasnet** [Chia-Network/chia-blockchain#10903] [Support]
Hello. I have a problem with DISC C, where I have instaled windows and the blockchain. The .chia folder is about 92GB I dont have more space on disc C. how can i decrease the size of this folder? Please HELP!!! (Sorry my English is not very good jeje)

**Comment by nurazhardotcom** [comment]
upgraded to db v2?

**Reply by nicolasnet** [reply]
I didnt upgrade to db V2, how can i do that?

**Reply by nurazhardotcom** [reply]
https://github.com/Chia-Network/chia-blockchain/wiki/CLI-Commands-Reference#upgrade

**Comment by loppefaaret** [comment]
if you look through some of the other support discussions on the same subject, there are some good guides to get passed this issue
https://github.com/Chia-Network/chia-blockchain/discussions/10845
